.github/workflows/release.yml

```diff
@@ -78,6 +79,11 @@ jobs:
     runs-on: ubuntu-24.04
 
     steps:
+    - name: Check access to GitHub secrets
+      run: echo "Pull request that trigger this workflow need to be on upstream repo to access GitHub secrets."; exit 1
+      # if a pull request is merged from a fork
+      if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'aiidateam'
```

Comment:
This workflow is only triggered by tag push so this will never be hit and is not needed here.